specification/migrate/reindex/MigrateReindexResponse.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AcknowledgedResponseBase } from '@_types/Base'
+
+export class Response {
+  body: AcknowledgedResponseBase
```

Comment:
Again, [toXContent](https://github.com/elastic/elasticsearch/blob/859f9c38b5aca969d894db0e79af93671b7a6e1c/x-pack/plugin/migrate/src/main/java/org/elasticsearch/xpack/migrate/action/ReindexDataStreamAction.java#L184-L192) does not appear to be used. (And this is confirmed by the YAML tests.)

Reply:
This is for historical reasons. The Response originally returned information, but was changed to return nothing other than what AcknowledgeResponse returns. I'll fix that in Elasticsearch.
The request implements toXContent, but that's for serialization testing in the unit test only.

Reply:
Removed in https://github.com/elastic/elasticsearch/pull/120321. I think this PR is correct as-is though (for this).